static/js/containers/DashboardPage_test.js

```diff
@@ -619,4 +621,40 @@ describe('DashboardPage', () => {
       });
     });
   });
+
+  describe("checkout", () => {
+    let dashboardResponse;
+    beforeEach(() => {
+      // Limit the dashboard response to 1 program
+      dashboardResponse = {"programs": [R.clone(DASHBOARD_RESPONSE.programs[0])]};
```

Comment:
Can you use `makeProgram()` instead? This should also let you use `program.courses[0]` for the course in the test below.